### PR TITLE
Story/ccls 1993 case costs

### PR DESCRIPTION
**Pull request by PhilDigitalJustice**

### CODE REVIEW

File: src/main/java/uk/gov/laa/ccms/caab/bean/validators/costs/CostDetailsValidator.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    validateRequiredField("requestedCostLimitation",
+        String.valueOf(costDetails.getRequestedCostLimitation()),
+        "Requested cost limitation", errors);
+
+    validateCurrencyField("requestedCostLimitation",
+        String.valueOf(costDetails.getRequestedCostLimitation()),
+        "Requested cost limitation", errors);
+
+    if (costDetails.getRequestedCostLimitation() != null
```
Comment (porritta) [inline src/main/java/uk/gov/laa/ccms/caab/bean/validators/costs/CostDetailsValidator.java:52]: Feels like upper limit validation will be used elsewhere. Could create an abstractvalidator method for it?

File: src/main/java/uk/gov/laa/ccms/caab/controller/HomeController.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package uk.gov.laa.ccms.caab.controller;
 
+import static uk.gov.laa.ccms.caab.constants.SessionConstants.APPLICATION_ID;
```
Comment (porritta) [inline src/main/java/uk/gov/laa/ccms/caab/controller/HomeController.java:3]: Not used?

Comment (PhilDigitalJustice) [reply src/main/java/uk/gov/laa/ccms/caab/controller/HomeController.java:3]: amended

File: src/main/java/uk/gov/laa/ccms/caab/bean/validators/costs/CostDetailsValidator.java (hunk 1 above)
Comment (PhilDigitalJustice) [reply src/main/java/uk/gov/laa/ccms/caab/bean/validators/costs/CostDetailsValidator.java:52]: amended

**Review by porritta — CHANGES_REQUESTED** [review]
Comments added

**Review by porritta — APPROVED** [review]
LGTM

**Pull request merged**